packages/wonder-blocks-data/src/hooks/__tests__/use-cached-effect.test.js

```diff
@@ -190,13 +229,137 @@ describe("#useCachedEffect", () => {
             expect(fakeHandler).toHaveBeenCalledTimes(1);
         });
 
+        it.each(
+            Array.from(FetchPolicy.members()).filter(
+                (e) => e !== FetchPolicy.CacheOnly,
+            ),
+        )(
+            "should provide function that causes refetch with FetchPolicy.%s",
+            async (fetchPolicy) => {
+                // Arrange
+                const response = Promise.resolve("DATA1");
+                const fakeHandler = jest.fn().mockReturnValue(response);
+
+                // Act
+                const {
+                    result: {
+                        current: [, refetch],
+                    },
+                } = clientRenderHook(() =>
+                    useCachedEffect("ID", fakeHandler, {fetchPolicy}),
+                );
+                fakeHandler.mockClear();
+                await act((): Promise<mixed> => response);
+                act(refetch);
+                await act((): Promise<mixed> => response);
+
+                // Assert
+                expect(fakeHandler).toHaveBeenCalledTimes(1);
```

Comment:
What's the connection between fakeHandler being called once and the test title?

Reply:
Just being explicit. When refetch is called, the handler should be called. Seems fair to also expect it to only invoke the handler once in that scenario.